kubernetes/cleanup

```diff
@@ -2,5 +2,11 @@
 
 log()   { echo ">>> $1" ; }
 
+[[ -d manifest-templates ]] || { echo >&2 "Please run this script from within the kubernetes folder"; exit 1; }
```

Comment:
we could also just cd into the folder, i guess it would be a better default behaviour